packages/Kokkos/test/tstLinearBVH.cpp

```diff
@@ -53,34 +71,43 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( LinearBVH, structured_grid, NO )
                 };
             }
 
-    DataTransferKit::Details::CollisionList collision_list;
     using DeviceType = typename DataTransferKit::BVH<NO>::DeviceType;
+    using ExecutionSpace = typename DeviceType::execution_space;
     Kokkos::View<DataTransferKit::AABB *, DeviceType, Kokkos::MemoryUnmanaged>
         bounding_boxes( bounding_boxes_vector.data(), n );
     DataTransferKit::BVH<NO> bvh( bounding_boxes );
 
     // (i) use same objects for the queries than the objects we constructed the
     // BVH
-    for ( int i = 0; i < n; ++i )
-        //    DataTransferKit::traverseRecursive(collision_list, bvh,
-        //    bvh._bounding_boxes[i], i, bvh.getRoot());
-        DataTransferKit::Details::traverseIterative( collision_list, bvh,
-                                                     bounding_boxes[i], i );
```

Comment:
maybe a comment here to say you will print the number of collisions detected (that you expect to be 1) in `identity[i][0]` and the index of the object it collided with in `identity[i][1]`.